### ISSUE TITLE
Reload JEI automatically after /mt reload is done

### ISSUE THREAD

**Issue by BBoldt** [jaredlll08/CraftTweaker#4]
```
<BBoldt> I have a suggestion, automatically reload JEI when the /mt reload command is done
<Jared> ugh
<Jared> it does?
<Jared> in my tests, jei reloaded fine
<BBoldt> it has to be done manually as far as I know
<Jared> or well the changes were implemented without needing to reload jei or anyinth
<BBoldt> yes
<BBoldt> but jei doesn't show the changed recipes until you reload it
<BBoldt> which can be done by turning on and turning off "dev mode"
<Jared> ah, I'll look more into it
<Jared> could you actually make an issue on the github
<BBoldt> yeah
<Jared> makes me feel special :3
```
I talked to @mezz about this, but I don't remember what he said.

**Comment by mezz** [comment]
Talked to Jared on IRC. I just added a method in `IJeiHelpers` called `reload()` that can be called when `/mt reload` is done.

https://github.com/mezz/JustEnoughItems/commit/9dbac3fefd2b20b26621ec0029c7056f602a467b

**Comment by jaredlll08** [comment]
Will be added when it is in a release

**Comment by mezz** [comment]
This has just been released in 2.14.0

**Comment by jaredlll08** [comment]
kk I'll add it next release
